packages/contracts/src/networks.ts

```diff
@@ -346,199 +390,262 @@ export const SPHINX_NETWORKS: Array<SupportedNetwork> = [
       apiURL: 'https://base-sepolia.blockscout.com/api',
       browserURL: 'https://base-sepolia.blockscout.com/',
       envKey: 'BASE_ETHERSCAN_API_KEY',
+      supported: true,
     },
     currency: 'ETH',
     dripSize: '0.15',
     requiredEnvVariables: ['ALCHEMY_API_KEY'],
     networkType: 'Testnet',
+    dripVersion: 0,
+  },
+  {
+    name: 'celo',
+    chainId: BigInt(42220),
+    rpcUrl: () =>
+      `https://celo-mainnet.infura.io/v3/${process.env.INFURA_API_KEY}`,
+    etherscan: {
+      apiURL: 'https://api.celoscan.io/api',
+      browserURL: 'https://celoscan.io/',
+      envKey: 'CELO_ETHERSCAN_API_KEY',
+      supported: true,
+    },
+    currency: 'CELO',
+    dripSize: '1',
+    requiredEnvVariables: ['INFURA_API_KEY'],
+    networkType: 'Mainnet',
+    dripVersion: 0,
+  },
+  {
+    name: 'celo_alfajores',
+    chainId: BigInt(44787),
+    rpcUrl: () =>
+      `https://celo-alfajores.infura.io/v3/${process.env.INFURA_API_KEY}`,
+    etherscan: {
+      apiURL: 'https://api-alfajores.celoscan.io/api',
+      browserURL: 'https://alfajores.celoscan.io/',
+      envKey: 'CELO_ETHERSCAN_API_KEY',
+      supported: true,
+    },
+    currency: 'CELO',
+    dripSize: '0.15',
+    requiredEnvVariables: ['INFURA_API_KEY'],
+    networkType: 'Testnet',
+    dripVersion: 0,
+  },
+  {
+    name: 'moonriver',
+    chainId: BigInt(1285),
+    rpcUrl: () => process.env.MOONRIVER_RPC_URL!,
+    etherscan: {
+      apiURL: 'https://api-moonriver.moonscan.io/api',
+      browserURL: 'https://moonriver.moonscan.io',
+      envKey: 'MOON_ETHERSCAN_API_KEY',
+      supported: true,
+    },
+    currency: 'MOVR',
+    dripSize: '0.15',
+    requiredEnvVariables: ['MOONRIVER_RPC_URL'],
+    dripVersion: 0,
+    networkType: 'Mainnet',
+  },
+  {
+    name: 'moonbeam',
+    chainId: BigInt(1284),
+    rpcUrl: () => process.env.MOONBEAM_RPC_URL!,
+    etherscan: {
+      // Seems like their API is bugged, I keep getting "Invalid API Key" when trying to verify.
+      // My API key is definitely valid.
+      apiURL: 'https://api-moonbeam.moonscan.io/api',
+      browserURL: 'https://moonbeam.moonscan.io',
+      envKey: 'MOON_ETHERSCAN_API_KEY',
+      supported: true,
+    },
+    currency: 'GLMR',
+    dripSize: '1',
+    requiredEnvVariables: ['MOONBEAM_RPC_URL'],
+    networkType: 'Mainnet',
+    dripVersion: 0,
+  },
+  {
+    name: 'moonbase_alpha',
+    chainId: BigInt(1287),
+    rpcUrl: () => process.env.MOONBASE_ALPHA_RPC_URL!,
+    etherscan: {
+      apiURL: 'https://api-moonbase.moonscan.io/api',
+      browserURL: 'https://moonbase.moonscan.io/',
+      envKey: 'MOON_ETHERSCAN_API_KEY',
+      supported: true,
+    },
+    currency: 'DEV',
+    dripSize: '0.05',
+    requiredEnvVariables: ['MOONBASE_ALPHA_RPC_URL'],
+    networkType: 'Testnet',
+    dripVersion: 1, // todo - redeploy this
```

Comment:
Moonbase alpha is down right now. I will come back to this when it comes back online.